hazelcast/src/main/java/com/hazelcast/config/ReplicatedMapConfig.java

```diff
@@ -97,13 +107,14 @@ public ReplicatedMapConfig setName(String name) {
     /**
      * The number of milliseconds after a put is executed before the value is replicated
      * to other nodes. During this time, multiple puts can be operated and cached up to be sent
-     * out all at once after the delay. This highers the latency for eventually consistency but lowers IO operations.
+     * out all at once after the delay.
      * Default value is 100ms before a replication is operated. If 0, no delay is used and
      * all values are replicated one by one.
      *
      * @return the number of milliseconds after a put is executed before the value is replicated
      * to other nodes.
      */
+    @Deprecated
```

Comment:
as I user when I see something is deprecated then I'd like see:
1. why?
2. what to used instead where applicable. if it's not applicable then it should be explained in the `why?` part.